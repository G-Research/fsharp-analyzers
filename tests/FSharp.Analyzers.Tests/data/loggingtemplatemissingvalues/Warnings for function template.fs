module M

    open Microsoft.Extensions.Logging

    let testlog () =
        use factory = LoggerFactory.Create(fun b -> b.AddConsole() |> ignore)
        let logger: ILogger = factory.CreateLogger("Program")
        let timeWasteString = "time waste"

        logger.LogWarning(id "Foo {Wasted}\n{Result}", timeWasteString)
