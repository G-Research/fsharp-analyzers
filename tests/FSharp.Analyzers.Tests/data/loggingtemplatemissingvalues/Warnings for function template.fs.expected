GRA-LOGTEMPLMISSVALS-001 | Warning | (10,8 - 10,71) | The given values in your call to ILogger.LogWarning don't match the expected templated args. | []
